src/System.IdentityModel.Tokens.Jwt/JwtSecurityTokenHandler.cs

```diff
@@ -1141,7 +1141,7 @@ private ClaimsPrincipal ValidateTokenPayload(JwtSecurityToken jwtToken, TokenVal
             if (validationParameters.SaveSigninToken)
                 identity.BootstrapContext = jwtToken.RawData;
 
-            LogHelper.LogInformation(TokenLogMessages.IDX10241, jwtToken.RawData);
+            LogHelper.LogInformation(TokenLogMessages.IDX10241, JwtTokenUtilities.PrepareSecurityArtifact(jwtToken.RawData));
```

Comment:
Could this be SecurityToken.ToString()?

Reply:
IMO neither of those is the ideal way. We want to delay string operations as log as possible. Especially for informational logs which may not ever actually see the light of day. Those would just be wasted CPU cycles.